### PR TITLE
Login

### PR DESCRIPTION
**Pull request by YogeshSharma0201**
Instead of redirecting to bot page send user a message that they are logged in.

### CODE REVIEW

File: server/server.ts [hunk 1]
```diff
@@ -45,9 +47,20 @@ export function startServer(db: Db) {
                     }
 
                     if (result.result.ok) {
-
                         res.status(200).write("OK");
-                        res.end();
+
+                        axios.post(`https://api.telegram.org/bot${process.env.TOKEN}/sendMessage`, {
```
Comment (ishanjain28) [inline server/server.ts]: We are already using [node-telegram-bot-api](http://npm.im/node-telegram-bot-api), Don't need to use the API directly.

Comment (YogeshSharma0201) [inline server/server.ts]: In that case i would have to config bot in main.ts and pass it to startbot and server just like db, should i do that? Or is there any other way?

Comment (ishanjain28) [inline server/server.ts]: @YogeshSharma0201 I changed it all a bit, Now, I create a `bot` class which extends Event Emitter, All the Event listeners are defined in `bot.ts` and a object of type `bot` is created and exported, In other modules like the server you can import `bot` from `bot.ts` and use it. This idea looked okay to me but if you hav a better idea, I would love to hear it. 

Also, Imgur login is complete, Do you want to work on image upload? or something else?

Comment (ishanjain28) [inline server/server.ts]: The aforementioned commit, is [here](https://github.com/ishanjain28/imgur-bot/commit/f588373996e382123a5a927395e6e278181b2b8c)

File: server/server.ts [hunk 2]
```diff
@@ -45,9 +50,19 @@ export function startServer(db: Db) {
                     }
 
                     if (result.result.ok) {
-
                         res.status(200).write("OK");
+                        axios.post(`https://api.telegram.org/bot${process.env.TOKEN}/sendMessage`, {
```
Comment (ishanjain28) [inline server/server.ts]: We are already using `node-telegram-bot-api`, Why are you using telegram's bot api directly?

**Pull request closed without merging**